report/report.go

```diff
@@ -298,8 +297,8 @@ func EnsureBackupVersionCompatibility(backupVersion string, restoreVersion strin
 	restoreSemVer, err := semver.Make(restoreVersion)
 	gplog.FatalOnError(err)
 	if backupSemVer.GT(restoreSemVer) {
-		gplog.Fatal(errors.Errorf("gprestore %s cannot restore a backup taken with gpbackup %s; please use gprestore %s or later.",
-			restoreVersion, backupVersion, backupVersion), "")
+		gplog.Warn("gprestore %s cannot restore a backup taken with gpbackup %s; please use gprestore %s or later.",
```

Comment:
Generally speaking, gpbackup is mainly used for database backup, not for upgrading, because different versions may have potential data compatibility issues.
If upgrading is needed, it is recommended to use other approaches.

Reply:
Yes, it is suggested.